spec/features/admin/index_spec.rb

```diff
@@ -10,12 +10,13 @@
           expect(page).to have_content("Admin Dashboard")
         end
       end
-
+      
       it "Then I see links to '/admin/merchants' and '/admin/invoices'" do
         merchant = create(:merchant) 
         expect(merchant.valid?).to be true
         visit "/admin"
-
+        
+        save_and_open_page
```

Comment:
Quick not, can we remove the save_and_open_page and push the PR again, just to keep those kind of diagnostic tools off main, thanks!

Reply:
Just updated that! Thanks for the reminder.